app/renderer/lib/appium-client.js

```diff
@@ -202,8 +201,23 @@ export default class AppiumClient {
 
   async getWindowUpdate () {
     let windowSize, windowSizeError;
+    const {client:{capabilities:{deviceScreenSize, platformName}}} = this.driver
     try {
-      windowSize = await this.driver.getWindowRect();
+      // The call doesn't need to be made for Android for two reasons
+      // - when appMode is hybrid Chrome driver doesn't know this this command
+      // - the data is already on the driver
+      //   this.driver.client.capabilities.deviceScreenSize => "{width}x{length}"
+      if (platformName.toLowerCase() === 'android') {
```

Comment:
can platformName be not a string?

Reply:
not sure about that, it comes back from the capabilities, should I do a type check here?

Reply:
I usually use lodash to make it safe: `_.lower(platformName) === 'android'`

Reply:
Done, thanks